src/test/java/steps/SpaceStep.java

```diff
@@ -30,8 +40,9 @@
      */
     @When("The user create a new space with the following name (.*)")
     public void createNewSpace(String nameSpace) {
-        starPage = new StarPage();
+        starPage = new SpacePage();
         starPage.addNewSpace(nameSpace);
```

Comment:
Please change the order between line 44 and line 45, context should be first.

Reply:
Done.

Reply:
Done.